cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/filters/PassticketFilterFactory.java

```diff
@@ -54,10 +46,16 @@ protected TicketResponse getResponseFor401() {
         return new TicketResponse();
     }
 
+    @Override
+    public String getEndpointUrl(ServiceInstance instance) {
+        return String.format(TICKET_URL, instance.getScheme(), instance.getHost(), instance.getPort(), instance.getServiceId().toLowerCase());
+    }
+
     @Override
     protected WebClient.RequestHeadersSpec<?> createRequest(ServiceInstance instance, String requestBody) {
+        String url = getEndpointUrl(instance);
```

Comment:
Is there a reason to override it? It looks the same as the abstract. But it is not easy to compare it in the browser.

Reply:
you are right, thanks